src/js/api/config.js

```diff
@@ -61,9 +61,39 @@ function _adjustDefaultBwEstimate(estimate) {
     return Defaults.bandwidthEstimate;
 }
 
+function _copyToLocalization(allOptions) {
+    const { advertising, related, sharing } = allOptions;
+    const localization = Object.assign({}, allOptions.localization);
+
+    if (advertising) {
+        localization.advertising = Object.assign({}, localization.advertising, advertising);
+    }
+
+    if (related) {
+        localization.related = localization.related || {};
+        if (typeof related === 'object') {
+            localization.related.heading = localization.related.heading || related.heading;
+            localization.related.autoplaymessage = localization.related.autoplaymessage || related.autoplaymessage;
+        } else {
+            localization.related.heading = localization.related.heading || localization.related;
```

Comment:
I think you might have overlooked one of my previous comments https://github.com/jwplayer/jwplayer/pull/3084#discussion_r221345406

Reply:
this would break if `localization.related` is a string